packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -355,18 +357,43 @@ export class GutenbergEditorPage {
 
 	/**
 	 * Opens the Nav Sidebar on the left hand side.
+	 *
+	 * On desktop sized viewport, this will open the editor block sidebar listing recently edited posts and drafts.
+	 *
+	 * On mobile sized viewport, this method will pass through.
+	 *
 	 */
 	async openNavSidebar(): Promise< void > {
 		const frame = await this.getEditorFrame();
-		await frame.click( selectors.openSidebarButton );
+		if ( getTargetDeviceName() === 'desktop' ) {
+			await frame.click( selectors.desktopEditorSidebarButton );
+		}
 	}
 
 	/**
-	 * Clicks on the Dashboard link within the Block Editor Sidebar.
+	 * Returns to the Posts > All Posts view in Calypso.
+	 *
+	 * On desktop sized viewport, this method clicks on the `< All Posts` link in the block editor sidebar.
+	 * Note, for desktop the editor sidebar must be open. To open the sidebar, call `openNavSidebar` method.
+	 *
+	 * On mobile sized viewport, this method clicks on Navbar > My Sites.
+	 *
+	 * For both cases the esulting page will be the `My Home` page.
 	 */
-	async returnToDashboard(): Promise< void > {
+	async returnToHomeDashboard(): Promise< void > {
```

Comment:
Optional: but I kind of feel like that function should take care of opening the nav sidebar for desktop if needed. It feels a little weird that people using this function have to include a call to `openNavSidebar` in their specs before it, but that only matters for the desktop case.